website/source/docs/auth/aws-ec2.html.md

```diff
@@ -275,7 +275,9 @@ $ vault auth-enable aws-ec2
 #### Configure the credentials required to make AWS API calls
 
 Note: the client uses the official AWS SDK and will use environment variable or
-IAM role-provided credentials if available. The AWS credentials used require the IAM action `ec2:DescribeInstances` to be allowed.
+IAM role-provided credentials if available. The AWS credentials used require
+the IAM actions `ec2:DescribeInstances` and `iam:GetInstanceProfile` to be
+allowed.
```

Comment:
@mfischer-zd Can you please add additional note as to *when* `iam:GetInstanceProfile` is required?